### ISSUE TITLE
delete signatures by the admin

### ISSUE THREAD

**Issue by nk-Zammad** [zammad/zammad#2360]
### Infos:

* Used Zammad version: 2.7 (and develop)
* Installation method (source, package, ..): n/a
* Operating system: n/a
* Database + version: n/a
* Elasticsearch version: n/a
* Browser + version: applies to all Browsers

### Expected behavior:

* it is expected that in the Admin Interface under Channels-> E-Mail-> Signature 
The signature can be deleted by the Customer himself. 

### Actual behavior:

* the actual behavior is that you can create signatures as an admin (under Channels-> E-Mail-> Signature ) but you can't delete them yourself.

### Steps to reproduce the behavior:

* i go to channels in the Admin Interface, click email and go to the signatures tab. There I want to be able to delete signatures myself.

Yes I'm sure this is a bug and no feature request or a general question.

**Comment by ghost** [comment]
Has not been fixed yet!

**Comment by MrGeneration** [comment]
@RidgeLock correct, the issue is still open. Nobody claimed it would have been fixed.
We see this as an enhancement which currently has no priority or roadmap.

Please note that we can't provide ETAs.
I'm locking the conversation of this issue to stop `+1` and similar bumps. This helps us to focus.
